extra/lib.sh

```diff
@@ -125,9 +126,17 @@ function letsencrypt_cert() {
     __mydomain=$__domain
   fi
 
-  /usr/bin/certbot-auto certonly -n --agree-tos --standalone --standalone-supported-challenges tls-sni-01 -m "$__myemail" -d "$__mydomain"
-  sudo ln -s "/etc/letsencrypt/live/$__mydomain/cert.pem" "$1" || true
-  sudo ln -s "/etc/letsencrypt/live/$__mydomain/privkey.pem" "$2" || true
+  if [[ $__docker = true ]]; then
+    echo "#!/bin/bash" >> /root/tmp/certbot.sh
```

Comment:
I was just making an attempt to build the fbctf image in prod successfully. I know the code above is nasty, and maybe you can make me the `certbot_helper.sh`, which I can use it to let the process of dockerizing more neatly.